content/community/changelog/app-nuget/v5/_index.en.md

```diff
@@ -0,0 +1,7 @@
+---
+title: v4
```

Comment:
```suggestion
title: v5
```

Reply:
The one thing I should have gotten right 😆 Thx :)